integration_tests/test_ibc_rly.py

```diff
@@ -57,24 +57,56 @@ def ibc(request, tmp_path_factory):
     )
 
 
+def amount_dict(amt, denom):
+    return [
+        AttributeDict(
+            {
+                "amount": amt,
+                "denom": denom,
+            }
+        )
+    ]
+
+
 def coin_received(receiver, amt, denom):
     return {
         "receiver": receiver,
-        "amount": [(amt, denom)],
+        "amount": [
+            AttributeDict(
```

Comment:
You seems overrided the amount_dict fix?

Reply:
yes, [amount](https://github.com/cosmos/cosmos-sdk/blob/v0.47.9/x/bank/types/events.go#L36) is coin type and turn to dict in new web3.py